geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -185,10 +202,17 @@ public LocatorClusterManagementService(InternalConfigurationPersistenceService p
     }
 
     // persist configuration in cache config
-
-    configurationManager.add(config, groupName);
-    result.setStatus(StatusCode.OK,
-        "Successfully updated configuration for " + groupName + ".");
+    if (groups.size() == 1) {
+      String groupName = groups.iterator().next();
+      configurationManager.add(config, groupName);
+      result.setStatus(StatusCode.OK,
+          "Successfully updated configuration for " + groupName + ".");
+    } else {
+      for (String groupName : groups) {
+        configurationManager.add(config, groupName);
+      }
+      result.setStatus(StatusCode.OK, "Successfully updated configuration");
+    }
```

Comment:
It appears the else branch covers would work for a size of `1` as well.

Reply:
yes, all elements nested under <region> should be RegionAware, this include index, lucene index and any other custom elements that will be nested under "region".